tests/unit/type/stream-cgroups.tcl

```diff
@@ -218,13 +218,40 @@ start_server {
         ]
         assert {[llength [lindex $reply 0 1 0 1]] == 2}
         assert {[lindex $reply 0 1 0 1] eq {a 1}}
+
+        set reply [
+            r XPENDING mystream mygroup - + 10
+        ]
+        assert {[llength $reply] == 1}
+        set reply [
+            r XPENDING mystream mygroup - + 10 client1
+        ]
+        assert {[llength $reply] == 1}
+        set reply [
+            r XPENDING mystream mygroup - + 10 client2
+        ]
+        assert {[llength $reply] == 0}
+
         r debug sleep 0.2
         set reply [
             r XCLAIM mystream mygroup client2 10 $id1
         ]
         assert {[llength [lindex $reply 0 1]] == 2}
         assert {[lindex $reply 0 1] eq {a 1}}
 
+        set reply [
+            r XPENDING mystream mygroup - + 10
+        ]
+        assert {[llength $reply] == 1}
+        set reply [
+            r XPENDING mystream mygroup - + 10 client1
+        ]
+        assert {[llength $reply] == 0}
+        set reply [
+            r XPENDING mystream mygroup - + 10 client2
+        ]
+        assert {[llength $reply] == 1}
+
```

Comment:
please add a short comment above the block stating what it does (i.e. check the PEL in both the consumer and group)
btw, why so many lines?

p.s. didn't you want to add a similar check in the new test you added? (making sure that when the consumer is the same one)